curs_main.c

```diff
@@ -1802,7 +1802,9 @@ int mutt_index_menu (void)
       case OP_MAIN_MODIFY_LABELS_THEN_HIDE:
       {
 	if (Context->magic != MUTT_NOTMUCH) {
-	  mutt_message (_("No virtual folder, aborting."));
+	  // remove remaining arguments
+	  mutt_get_field ("", buf, sizeof(buf), MUTT_NM_TAG);
```

Comment:
I don't think using `mutt_get_field()` is necessary. If I'm not wrong, it's creating a useless input for the user, and in this case without any prompt.

The way the buffer is discarded by all functions is to do:

```
buf[0] = '\0';
```